app/containers/Communities/index.js

```diff
@@ -59,11 +63,16 @@ export const Communities = ({
           SubHeader={SubHeader}
           changeSorting={changeSorting}
           sorting={sorting}
-          communitiesNumber={communities?.length ?? 0}
+          communitiesNumber={unhiddenCommunities?.length ?? 0}
```

Comment:
leave just communities, no need to rename

Reply:
everywhere

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1467/commits/ad29cb625792e7bd3c3688a2c46d50d6dc3110c1